src/components/MySpace/MySpace.tsx

```diff
@@ -145,6 +171,23 @@ const MySpace = ({ bookmarks }: { bookmarks: BookmarkRecords }) => {
                 )
               }
 
+              if (
+                isFeaturedShortcuts(widget) &&
+                flags &&
+                flags?.featuredShortcuts
```

Comment:
Suggestion(nit-pick): Feels redundant to check `flags` and have `flags?.featuredShortcuts` in the same if condition. Probably could remove line 176. Though I see it is in the above GuardianIdeal checks too.